lib/src/builder.dart

```diff
@@ -98,7 +99,11 @@ class GeneratorBuilder extends Builder {
 
 Stream<GeneratedOutput> _generate(LibraryElement unit,
     List<Generator> generators, BuildStep buildStep) async* {
-  for (var element in getElementsFromLibraryElement(unit)) {
+  var elements = safeIterate(getElementsFromLibraryElement(unit), (e, [_]) {
+    log.severe('Failed to resolve ${buildStep.inputId}.');
```

Comment:
It could mean a lot of things which is why I kept it so general. Up until now we encounter this mostly because of our bazel codegen not correctly handling things - quirks in build targets etc.

I could add something like "File an issue" but the details of how and where to file will likely change by situation...